src/game-board/game-board.html

```diff
@@ -90,8 +90,7 @@
       */
       _cardDropped(e){
 
-        if(this.movingStack.stack.cards[this.movingStack.stack.cards.length-1].rank === this.movingCard.rank
-          && this.movingStack.stack.cards[this.movingStack.stack.cards.length-1].suit === this.movingCard.suit){
+        if(this._canCardMoveHere(e.detail.stack.cards[e.detail.stack.cards.length -1].rank, this.movingCard.rank)){
```

Comment:
This is really verbose. I'd prefer to pull the stack out into its own variable for reuse:

    const {stack} = e.detail;
    
    if (this._canCardMoveHere(stack.cards[stack.cards.length - 1].rank, this.movingCard.rank) {
      // ...
    }

Also are we sure that `e.detail.stack.cards` will always contain at least one value? For example, because we get the last value like `cards[cards.length - 1]` if the `length` were ever zero this would try to get the `-1` index which would be undefined and then when you try to access `rank` on it you get an error.